pkg/seccomp/seccomp_test.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2018 The gVisor Authors.
+// Copyright 2020 The gVisor Authors.
```

Comment:
pls don't update the year for existing files.

Reply:
Ok